models/containerized_model/DEPLOYMENT.md

```diff
@@ -38,6 +39,8 @@ spec:
       env:
         - name: MODEL_PATH
           value: "s3://bucket/model.pkl"
+        - name: METADATA_PATH
+          value: "s3://bucket/metadata.yml"
         - name: BUCKET_ENDPOINT
           value: "<your-s3-bucket-endpoint>"
```

Comment:
is the bucket endpoint generic for diff. cloud providers like azure or limited to S3 at this time?